index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@1,900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Roboto:ital,wght@1,900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Roboto:ital,wght@1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="main-container">
+    <header class="logo">
+      <h1 class="me">Habtamu</h1>
+      <nav class="humberger">
+      <a href="#"><img class="vec" src="images/Union.png" alt=""></a>
+    </nav>
+  </header>
+    <section class="initial">
+      <h1 class="information">Hey There.
+      I’m Habtamu </h1>
+      <h2>I’am a Software Developer</h2>
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+      <ul class="icons">
+        <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/Linkedin.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/H.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/M.svg" alt=""></a></li>
+      </ul>
+    
+   <!--  <div class="bg">
+      <img id="back" src="images/Illustration-header.png">
+    </div> -->
```

Comment:
- [ ] As requested by the previous reviewer, kindly remove all commented code in your code base. :wink: